src/Magento/FunctionalTestingFramework/Test/Util/ActionMergeUtil.php

```diff
@@ -149,7 +149,7 @@ private function actionAttributeContainsSecretRef($actionAttributes)
                 return $this->actionAttributeContainsSecretRef($actionAttribute);
             }
 
-            preg_match_all("/{{_CREDS\.([\w]+)}}/", $actionAttribute, $matches);
+            preg_match_all("/{{_CREDS\.([\w|\/]+)}}/", $actionAttribute, $matches);
```

Comment:
Can we change this to a class constant? That way we can reuse it in TestGenerator.

Reply:
Done